src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -779,4 +780,56 @@ public void testYouCantWriteIntoAnExistingDirectory(){
         final String workspace = createTempDir("workspace").getAbsolutePath();
         writeToGenomicsDB(LOCAL_GVCFS, INTERVAL, workspace, 0, false, 0, 1);
     }
+
+    private void cleanupGCSFolder(String path) {
+        try {
+            if (BucketUtils.isCloudStorageUrl(path)) {
+                Files.list(Paths.get(path)).forEach(f -> {
+                    try {
+                        Files.deleteIfExists(Paths.get(f.toString()));
+                    } catch (DirectoryNotEmptyException e1) {
+                        cleanupGCSFolder(f.toString());
+                    } catch (IOException e) {
+                        // Ignore for now
+                    }
+                });
+                Files.deleteIfExists(BucketUtils.getPathOnGcs(path));
+            }
+        } catch (IOException e) {
+            // Ignore for now.
+        }
+    }
+
+    @Test(groups = {"bucket"})
+    public void testWriteToAndQueryFromGCS() throws IOException {
+        String workspace = BucketUtils.randomRemotePath(getGCPTestInputPath(), "","");
+        try {
+            Assert.assertNotNull(getGoogleServiceAccountKeyPath());
```

Comment:
You don't need this check (or the `UserException` catch block) in your test -- tests that are part of the "bucket" group only get run in travis when GOOGLE_APPLICATION_CREDENTIALS is already set.